### PR TITLE
 Updating deprecated scripts in McAfee & CrowdStrike playbooks  

### PR DESCRIPTION
**Pull request by EliorKedar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25297

## Description
Updating deprecated scripts in McAfee & CrowdStrike playbooks

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by yaron-libman** [comment]
Reviewed.

File: Packs/McAfee_DXL/Playbooks/playbook-Enrich_McAfee_DXL_using_3rd_party_sandbox_v2.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    timertriggers: []
+    ignoreworker: false
+  "7":
+    id: "7"
+    taskid: bfffbaa1-a550-47bd-8e84-e6c941cde205
+    type: playbook
+    task:
+      id: bfffbaa1-a550-47bd-8e84-e6c941cde205
+      version: -1
+      name: "WildFire - Detonate file"
```
Comment (michalgold) [inline Packs/McAfee_DXL/Playbooks/playbook-Enrich_McAfee_DXL_using_3rd_party_sandbox_v2.yml:139]: please change to the generic detonate file

File: Packs/McAfee_DXL/Playbooks/playbook-Enrich_DXL_with_ATD_verdict_v2.yml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    timertriggers: []
+    ignoreworker: false
+  "5":
+    id: "5"
+    taskid: c50351fa-76f1-43ce-852a-de0c66d9535b
+    type: playbook
+    task:
+      id: c50351fa-76f1-43ce-852a-de0c66d9535b
+      version: -1
+      name: "ATD - Detonate File"
```
Comment (michalgold) [inline Packs/McAfee_DXL/Playbooks/playbook-Enrich_DXL_with_ATD_verdict_v2.yml:107]: Please check the box for it to skip playbook if it's not there.

File: Packs/epo/Playbooks/playbook-McAfee_ePO_Repository_Compliance_v2.yml [hunk 3]
```diff
@@ -0,0 +1,815 @@
...
+    timertriggers: []
+    ignoreworker: false
+  "11":
+    id: "11"
+    taskid: dd5d9578-ca9e-4a3b-8088-940627f3fe9e
+    type: regular
+    task:
+      id: dd5d9578-ca9e-4a3b-8088-940627f3fe9e
+      version: -1
+      name: epo-get-current-dat
```
Comment (michalgold) [inline Packs/epo/Playbooks/playbook-McAfee_ePO_Repository_Compliance_v2.yml:214]: Change task name

File: Packs/CrowdStrikeHost/ReleaseNotes/1_1_1.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+- __FalconHost__
+Added support for 3 commands from the Threat graph API:
+  - ***cs-threatgraph-summary***
+  - ***cs-threatgraph-processes***
+  - ***cs-threatgraph-detections***
+#### Playbooks
+ - added rapid ioc hunting v2 playbook and replaced deprecated scripts.
```
Comment (michalgold) [inline Packs/CrowdStrikeHost/ReleaseNotes/1_1_1.md:9]: added should be with upper case A.

**Pull request merged**